doc/CHEATSHEET.md

```diff
@@ -158,13 +158,16 @@ txSkelTemplate
 
 ### Pay (transaction output)
 
-* `paysPK (wallet 3) (ada 6)`
-* `paysScript fooTypedValidator FooTypedDatum (ada 6)`
+* A simple value to a wallet: ```wallet 3 `receives` Value (ada 3)```
+* A value and an inline datum to a script: ```fooTypedValidator `receives` (InlineDatum FooTypedDatum <$$> Value (myToken 4 <> lovelace 160_000))```
+* Hashed datums (visible to the transaction or hidden from it): `... <$$> (VisibleHashedDatum dat)` or `... <$$> (HiddenHashedDatum dat)`
```

Comment:
Is `<$$>` a standard Haskell operator? And what's the difference with `<&&>` used in the example above?

Reply:
Good catch, I just mistakenly wrote <$$> in the CHEATSHEET file.